app/src/main/java/vn/sun/music_68/ui/home/recommend/RecommendPresenter.java

```diff
@@ -19,14 +19,15 @@ public RecommendPresenter(RecommendContract.View view, TrackRepository repositor
     public void getTrack(String api) {
         mRepository.getTracks(api, new TrackDataSource.DataCallback<Track>() {
             @Override
-            public void onSuccess(List<Track> datas) {
-
+            public void onSuccess(List<Track> data) {
+                mView.onLoadSuccess(data);
```

Comment:
chỗ này thì nên gọi là 
```suggestion
                mView.onShowTrack(data);
```
hay gì đó, vì View chỉ đảm nhận việc show hide data thôi chứ không liên quan gì đến việc load dữ liệu cả.